src/screens/Notifications/NotificationScreen.tsx

```diff
@@ -67,20 +71,48 @@ export class NotificationScreen extends Component<Props> {
       },
     });
 
-  goToConfirmScreen = () =>
-    this.props.navigation.navigate(Route.ConfirmEmail, {
-      email: this.props.email!,
-      flowType: ConfirmAddressFlowType.DELETE_ADDRESS,
+  unsubscribeAndRemoveEmail = () =>
+    this.props.navigation.navigate(Route.ChooseWalletsForNotification, {
+      flowType: ConfirmAddressFlowType.UNSUBSCRIBE,
+      subtitle: i18n.wallets.details.unsubscribeWallet,
+      description: i18n.notifications.chooseWalletsToUnsubscribeDescription,
+      email: this.props.email,
+      onSuccess: () => {
+        this.props.deleteNotificationEmail();
+        this.goToDeleteSuccessScreen();
+      },
+      walletsToSubscribe: this.props.subscribedWallets,
```

Comment:
rename to  `walletsToSubscribe` to `wallets`, we need a general name because some wallets on some flow ChooseWalletsForNotification are for subscription and some for unsubscription